be/src/olap/block_column_predicate.h

```diff
@@ -81,6 +81,12 @@ class BlockColumnPredicate {
         LOG(FATAL) << "should not reach here";
         return true;
     }
+
+    virtual bool evaluate_and(const StringRef* dict_words, const size_t dict_num) const {
```

Comment:
why not pass in std::vector<StringRef>&

Reply:
Cause `dict_words` is from `std::unique_ptr<StringRef[]> _dict_word_info` of `FileColumnIterator`